examples/04-Retrieval-Model.ipynb

```diff
@@ -0,0 +1,1847 @@
+{
```

Comment:
<p>We have ops <code>TagAsUserFeatures</code> and <code>TagAsItemFeatures</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>